linera-views/src/memory.rs

```diff
@@ -43,11 +47,18 @@ pub const TEST_MEMORY_MAX_STREAM_QUERIES: usize = 10;
 /// The analog of the database is the BTreeMap
 pub type MemoryStoreMap = BTreeMap<Vec<u8>, Vec<u8>>;
```

Comment:
Why `pub`? Please let's minimize the number of public objects in our libraries.

Reply:
Yes, it was possible to remove it. I thought `MemoryStoreMap` was part of the `MemoryStore` and so it had to be public. But maybe it was fine because it was just an alias.